apis/python/src/tiledbsoma/_dataframe.py

```diff
@@ -133,6 +147,14 @@ def index_column_names(self) -> Tuple[str, ...]:
         """
         return self._tiledb_dim_names()
 
+    @property
+    def domain(self) -> Tuple[Tuple[Any, Any], ...]:
+        """
```

Comment:
Is a separate "returns" section needed when the text of the docstring describes the returned value?

Reply:
good call - the style guide says about the Returns docstring section:

> It may also be omitted if the docstring starts with Returns or Yields (e.g. """Returns row from Bigtable as a tuple of strings.""") and the opening sentence is sufficient to describe the return value.


[Reference](https://google.github.io/styleguide/pyguide.html#doc-function-returns)